Title Tableau: Adidas Profit Analysis

Tableau Link: https://public.tableau.com/views/Adidas_Sales_17224995182660/Dashboard1?:language=en-US&:sid=&:redirect=auth&:display_count=n&:origin=viz_share_link


# Introduction
=============================

**Milestone 1**
<p style="font-size:18px;">
Made By  : Brenda Kwan<br>
Objective: This program was developed to analyze the factors that influence Adidas product sales in 2020-2021 to help Adidas achieve Adidas' goal of increasing profits by 150% in one year.
</p>



# 1) Identifying the Problem

## Problem Topic:
Adidas wants to increase profits by 150% in one year, because in the previous year (January 2020-December 2020), there was an increase in operating profit of 150% (can be seen in the line chart trend in the data analysis section) but they do not know what strategy to implement to achieve this goal.

## Industry Dataset:

Adidas Sales Data 2020-2021

Categories: Sportswear, Sales

Source of Data: https://www.kaggle.com/datasets/heemalichaudhari/adidas-sales-dataset

## Background Problem:
The company Adidas wants to increase profits by 150% in one year by hiring data analysts to analyze factors such as price per unit, time period, state, units sold, product category, costs of goods sold (COGS) and retailer.


## SMART Framework:

### SPECIFIC:
    Adidas wants to increase profits by understanding the factors that influence total sales and operating profit, including the highest and lowest profit margin products, retailers, most effective sales methods, and sales trends by region and time period

### MEASURABLE:
    Adidas wants to increase profits by 150% (as previously achieved) using effective strategies that are in line with data analysis of price per unit, time period, state, units sold, product category, costs of goods sold (COGS) and retailers

### ACHIEVABLE:
    Using descriptive and inferential statistical analysis and data visualization, Adidas can identify patterns and trends that will help guide sales and marketing strategies to achieve the desired profit increase

### RELEVANT:
    This project will help Adidas optimize sales and marketing strategies that are in line with the company's goals of increasing profits and expanding Adidas' market share

### TIME-BOUND:
    Achieve a 150% increase in profits within one year, with progress evaluated every three months

## Problem Statement:

Adidas wants increase its revenue by 150% in one year by identifying and analyzing the key factors that influence sales and profitability. This analysis will cover sales by product, region, and sales method and monthly sales trends. Using statistical analysis and data visualization, I will provide recommendations for optimal strategies that should be implemented by the sales and marketing management team with the company's goal of increasing profit and expanding market share ultimately

## Problem Description:

Title: To find out the factors that influence sales at Adidas to increase profit by 150% in 2020-2021

### Data Visualization:
1. Which region generates the highest and lowest total sales?
    - Map
    - Columns: state, total sales
    - Objective: to find out which regions in the US have the highest and lowest total sales, in order to manage inventory efficiently

2. What is the monthly trend of operating profit in the past year?
    - Line Chart
    - Columns: invoice date, operating profit
    - Objective: to find out if there is a seasonal pattern, identify periods of higher and lower profitability in order to make strategic decisions to optimize profitability next year

3. Does more units sold mean higher operating profit for different product categories?
    - Bubble Chart
    - Columns: product category, units sold, operating profit
    - Objective: to find out which Adidas product categories have the most sales and find out if there are other factors that impact profit besides units sold (e.g., lower profit even though many units sold, means the price is too low/discount is too high)

4. Which retailers contribute the most and the least to Adidas' operating profit?
    - Bar Chart
    - Columns: retailers, operating profit
    - Objective: to find out which retailers contribute the most and the least to Adidas' profit, so that Adidas can allocate more products to retailers that contribute significantly to Adidas' profit and for retailers that contribute low to profit, Adidas' management team can find out the reasons for poor performance and consider strategies to increase profit.

### Descriptive Statistics:

5. What is the correlation value of units sold and costs of goods sold (COGS)?
    - Spearman Correlation (skewed)
    - Column: units sold, COGS (costs of goods sold)
    - Purpose: to check whether operational, material, labor costs and transportation costs, etc. are high. If so, can suggest Adidas a strategy to reduce the COGS

6. What is the correlation value of units sold and price per unit?
    - Spearman Correlation (skewed)
    - Column: units sold, price per unit
    - Purpose: to find out if price per unit has an impact on units sold (e.g., if it is cheaper, more units are sold or not) to help Adidas implement a discount or price reduction strategy if there is a negative correlation

### Inferential Statistics:

7. Is there a significant difference in the average price per unit between high-selling and low-selling products?
    - Mann-Whitney U Test (skewed)
    - Column: price per unit, units sold
    - Purpose: to find out if the price of a product can affect units sold, and to find out if the price is competitive or not to help the Adidas management team in pricing strategy
    - **H0: The distribution of prices per unit of high-selling and low-selling products is the same and there is no difference in the central tendency of the price per unit between high-selling and low-selling products.**
    - **H1: The distribution of prices per unit of high-selling and low-selling products is different, there is a difference in the central tendency of the price per unit between high-selling and low-selling products**

# 2) Data Loading

In [1]:
# Import pandas package to create dataframe
import pandas as pd

# Read data from adidas_sales excel file and store data in dataframe
df = pd.read_excel('adidas_sales.xlsx')
df

Retailer  Retailer ID Invoice Date     Region          State  \
0     Foot Locker      1185732   2020-01-01  Northeast       New York   
1     Foot Locker      1185732   2020-01-02  Northeast       New York   
2     Foot Locker      1185732   2020-01-03  Northeast       New York   
3     Foot Locker      1185732   2020-01-04  Northeast       New York   
4     Foot Locker      1185732   2020-01-05  Northeast       New York   
...           ...          ...          ...        ...            ...   
9643  Foot Locker      1185732   2021-01-24  Northeast  New Hampshire   
9644  Foot Locker      1185732   2021-01-24  Northeast  New Hampshire   
9645  Foot Locker      1185732   2021-02-22  Northeast  New Hampshire   
9646  Foot Locker      1185732   2021-02-22  Northeast  New Hampshire   
9647  Foot Locker      1185732   2021-02-22  Northeast  New Hampshire   

            City                    Product  Price per Unit  Units Sold  \
0       New York      Men's Street Footwear              50        1200   
1       New York    Men's Athletic Footwear              50        1000   
2       New York    Women's Street Footwear              40        1000   
3       New York  Women's Athletic Footwear              45         850   
4       New York              Men's Apparel              60         900   
...          ...                        ...             ...         ...   
9643  Manchester              Men's Apparel              50          64   
9644  Manchester            Women's Apparel              41         105   
9645  Manchester      Men's Street Footwear              41         184   
9646  Manchester    Men's Athletic Footwear              42          70   
9647  Manchester    Women's Street Footwear              29          83   

      Total Sales  Operating Profit  Operating Margin Sales Method  
0        600000.0         300000.00              0.50     In-store  
1        500000.0         150000.00              0.30     In-store  
2        400000.0         140000.00              0.35     In-store  
3        382500.0         133875.00              0.35     In-store  
4        540000.0         162000.00              0.30     In-store  
...           ...               ...               ...          ...  
9643       3200.0            896.00              0.28       Outlet  
9644       4305.0           1377.60              0.32       Outlet  
9645       7544.0           2791.28              0.37       Outlet  
9646       2940.0           1234.80              0.42       Outlet  
9647       2407.0            649.89              0.27       Outlet  

[9648 rows x 13 columns]

# 3) Data Cleaning

Firstly, we need to check for missing values

In [2]:
# Display a summary of the dataframe
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 9648 entries, 0 to 9647
Data columns (total 13 columns):
 #   Column            Non-Null Count  Dtype         
---  ------            --------------  -----         
 0   Retailer          9648 non-null   object        
 1   Retailer ID       9648 non-null   int64         
 2   Invoice Date      9648 non-null   datetime64[ns]
 3   Region            9648 non-null   object        
 4   State             9648 non-null   object        
 5   City              9648 non-null   object        
 6   Product           9648 non-null   object        
 7   Price per Unit    9648 non-null   int64         
 8   Units Sold        9648 non-null   int64         
 9   Total Sales       9648 non-null   float64       
 10  Operating Profit  9648 non-null   float64       
 11  Operating Margin  9648 non-null   float64       
 12  Sales Method      9648 non-null   object        
dtypes: datetime64[ns](1), float64(3), int64(3), object(6)
memory usage: 980.0+ KB


Since all non-null counts of each column are the same, there are no missing values.

In [3]:
# Drop columns not used for data analysis
df = df.drop(columns=['Retailer ID', 'Sales Method', 'Region', 'City', 'Operating Margin'])
df

Retailer Invoice Date          State                    Product  \
0     Foot Locker   2020-01-01       New York      Men's Street Footwear   
1     Foot Locker   2020-01-02       New York    Men's Athletic Footwear   
2     Foot Locker   2020-01-03       New York    Women's Street Footwear   
3     Foot Locker   2020-01-04       New York  Women's Athletic Footwear   
4     Foot Locker   2020-01-05       New York              Men's Apparel   
...           ...          ...            ...                        ...   
9643  Foot Locker   2021-01-24  New Hampshire              Men's Apparel   
9644  Foot Locker   2021-01-24  New Hampshire            Women's Apparel   
9645  Foot Locker   2021-02-22  New Hampshire      Men's Street Footwear   
9646  Foot Locker   2021-02-22  New Hampshire    Men's Athletic Footwear   
9647  Foot Locker   2021-02-22  New Hampshire    Women's Street Footwear   

      Price per Unit  Units Sold  Total Sales  Operating Profit  
0                 50        1200     600000.0         300000.00  
1                 50        1000     500000.0         150000.00  
2                 40        1000     400000.0         140000.00  
3                 45         850     382500.0         133875.00  
4                 60         900     540000.0         162000.00  
...              ...         ...          ...               ...  
9643              50          64       3200.0            896.00  
9644              41         105       4305.0           1377.60  
9645              41         184       7544.0           2791.28  
9646              42          70       2940.0           1234.80  
9647              29          83       2407.0            649.89  

[9648 rows x 8 columns]

We have to rename 'Product' to 'Product Category' because 'Product' is not specific

In [4]:
# Rename column 'Product' to 'Product Category'
df = df.rename(columns = {'Product':'Product Category'})
df

Retailer Invoice Date          State           Product Category  \
0     Foot Locker   2020-01-01       New York      Men's Street Footwear   
1     Foot Locker   2020-01-02       New York    Men's Athletic Footwear   
2     Foot Locker   2020-01-03       New York    Women's Street Footwear   
3     Foot Locker   2020-01-04       New York  Women's Athletic Footwear   
4     Foot Locker   2020-01-05       New York              Men's Apparel   
...           ...          ...            ...                        ...   
9643  Foot Locker   2021-01-24  New Hampshire              Men's Apparel   
9644  Foot Locker   2021-01-24  New Hampshire            Women's Apparel   
9645  Foot Locker   2021-02-22  New Hampshire      Men's Street Footwear   
9646  Foot Locker   2021-02-22  New Hampshire    Men's Athletic Footwear   
9647  Foot Locker   2021-02-22  New Hampshire    Women's Street Footwear   

      Price per Unit  Units Sold  Total Sales  Operating Profit  
0                 50        1200     600000.0         300000.00  
1                 50        1000     500000.0         150000.00  
2                 40        1000     400000.0         140000.00  
3                 45         850     382500.0         133875.00  
4                 60         900     540000.0         162000.00  
...              ...         ...          ...               ...  
9643              50          64       3200.0            896.00  
9644              41         105       4305.0           1377.60  
9645              41         184       7544.0           2791.28  
9646              42          70       2940.0           1234.80  
9647              29          83       2407.0            649.89  

[9648 rows x 8 columns]

# 4) Analysis and Calculation

## Regions With the Highest and Lowest Total Sales
    - Data Visualisation: Chloropleth Map
    - Columns: state, total sales
    - Aim: to find out which areas in the US have the highest and lowest sales, so that Adidas can manage inventory more efficiently.

First, I will analyze the total sales per state to check which areas have the highest and lowest demand so I can target the areas with the highest demand and use strategies to increase demand in Nebraska to achieve the goal of increasing profits.

To make it easier to check Total Sales per state, I use a choropleth map to visualize the Total Sales data per state.

In [5]:
import plotly.express as px
import requests
import json

# Create a new dataframe for store operating profit per state
# Use sum to add up the operating profit for each state
df_grouped = df.groupby('State')['Total Sales'].sum().reset_index()

# URL to a GeoJSON file containing geographic data for the US states
geojson_url = 'https://raw.githubusercontent.com/PublicaMundi/MappingAPI/master/data/geojson/us-states.json'

# Create a 'response' object to make a 'get' request to retrieve GeoJSON data from a URL.
response = requests.get(geojson_url)

# Parse JSON 'response' into a Python dictionary
geojson = response.json()

# Create choropleth map using Plotly Express
fig = px.choropleth_mapbox(
    df_grouped,
    geojson=geojson,
    locations='State',
    # In the GeoJSON file, the state name can be found in the properties dictionary with the key name
    featureidkey='properties.name',
    color='Total Sales',
    color_continuous_scale='Viridis',
    # This parameter sets the Mapbox map style, carto-positron = light-themed map style
    mapbox_style="carto-positron", 
    zoom=3,
    # To center the map in the United States
    center={"lat": 37.0902, "lon": -95.7129},
    title='Total Sales by State'
)

# Center align map title
fig.update_layout(title_x=0.5)

# Display map
fig.show()

To check whether the total sales of Adidas per state is correct or not, I tested adding up the total sales for each state with sum. I used the states 'New York' and 'Nebraska' to check.

In [6]:
# To find the total of 'Total Sales' in New York as a test if the value in the map is correct or not
ny_sales = df[df['State'] == 'New York']['Total Sales'].sum()
print(ny_sales)

64229039.0


In [7]:
# To find the total of 'Total Sales' in Nebraska as a test if the value in the map is correct or not
nebraska_sales = df[df['State'] == 'Nebraska']['Total Sales'].sum()
print(nebraska_sales)

5929038.0


The above value when compared with the one on the map is the same, for New York the total sales are 64.23M which is the same as 64229039.0. For Nebraska, the total sales are 5.93M which is the same as the sum value using sum() which is 5929038.0

Based on the chloropleth map above, light yellow has the highest total sales while black or dark purple indicates the lowest total sales.

It can be seen in New York (light yellow), Adidas has the highest total sales, while in Nebraska, Adidas has the lowest total sales (darkest purple). This shows that in New York, demand for Adidas products is high while demand for Adidas products in Nebraska is low. With this information, to achieve the goal of increasing operating profit, there are several strategies that Adidas can do:
1. Conducting a more intensive marketing campaign in New York to maintain and increase consumer interest in Adidas products
2. Ensuring that products in demand are always available in New York to meet high demand
3. For New York and the yellow areas, Adidas needs to increase efforts in loyalty programs to retain customers such as exclusive discounts for Adidas members so that they can help strengthen customer relationships and increase retention
4. In Nebraska, Adidas can avoid overstocking by adjusting the amount of inventory based on actual demand. This will help reduce storage costs and the risk of unsold products.
5. Try offering discounts, price reductions or more product advertising to attract more consumers in Nebraska

In addition, the blue and green areas indicate significant market potential and it can be a great opportunity for Adidas to optimize its marketing and sales strategy. With a strategy to advertise more products or special discounts to attract consumers in these areas, Adidas can increase total sales in these areas and it will increase total sales overall

To make sure the state that has the highest and lowest total sales is correct, I use min() and max() to check

In [8]:
# Group by 'State' and add 'Total Sales' and create new dataframe for store total sales by state
df_grouped = df.groupby('State')['Total Sales'].sum().reset_index()

# Find the state with the highest total sales
max_sales_state = df_grouped.loc[df_grouped['Total Sales'].idxmin()]
print("Highest total sales in state", max_sales_state['State'], "with total Sales", max_sales_state['Total Sales'])

# Find the state with the lowest total sales
min_sales_state = df_grouped.loc[df_grouped['Total Sales'].idxmin()]
print("Lowest total sales in state", min_sales_state['State'], "with total Sales", min_sales_state['Total Sales'])

Highest total sales in state Nebraska with total Sales 5929038.0
Lowest total sales in state Nebraska with total Sales 5929038.0


## Viewing monthly operating profit trends over the past year
    - Data Visualisation: Line Chart
    - Columns: invoice date, operating profit
    - Purpose: to find out if there is a seasonal pattern, identify periods of higher and lower profitability in order to make strategic decisions to optimize profitability next year

- To see the trend of Adidas' operating profit in one year (2020-2021), because the operating profit data is continuous and because I want to see changes in operating profit over time to find patterns easily, I use a line chart.

In [9]:
# Copy the DataFrame and format 'Invoice Date' to '%Y-%m'
df_temp = df.copy()

# Convert the 'Invoice Date' column in your DataFrame to a string format of year and month
df_temp['Invoice Date'] = df_temp['Invoice Date'].dt.strftime('%Y-%m')

# Create a new dataframe for store operating profit per month
# Use sum to add operating profit per month
df_sales_date = df_temp.groupby(by='Invoice Date')[['Operating Profit']].sum()
df_sales_date.reset_index(drop=False, inplace=True)

# Sort values ​​by 'Invoice Date'
df_sales_date.sort_values(by='Invoice Date', ascending=True, inplace=True)

print(df_sales_date.head())

# Create a line chart using Plotly Express to see Adidas's profit trend every month in 2020-2021
fig = px.line(df_sales_date, x="Invoice Date", y="Operating Profit", title='Adidas Operating Profit Trend (2020-2021)')
fig.update_xaxes(tickangle=-90)
fig.update_layout(title_x=0.5)
fig.show()

  Invoice Date  Operating Profit
0      2020-01        5830005.41
1      2020-02        5207354.44
2      2020-03        5862006.26
3      2020-04        9301293.34
4      2020-05        5870841.95


For both 2020 and 2021, the following months operating profit increased:
- March - April
- June - July

For both 2020 and 2021, the following months operating profit decreased:
- January - February
- August - October

Peak Sales:
- July 2021 (summer)
- December 2021 (end of year/Christmas)

Based on the information, in March-April, operating profit increased possibly because it was the Easter season (March 22-April 25) and April Fools' Day (April 1), events widely celebrated by Americans. Celebrations such as Easter are often followed by shopping activities, gifts, and celebrations that can affect product sales. In June-July, operating profit increased possibly because it was the summer season in America, where summer break is a vacation from June to August. During this holiday period, there is an increase in shopping activities, including popular summer products, which can contribute to increased sales and profits. This indicates that there is an increase in demand during these periods and can be utilized by Adidas. For the following year, Adidas can launch marketing campaigns targeted at certain celebrations or seasons. For example, offer special discounts during the festive period to increase total sales.

But in January-February, operating profit decreases: after the year-end holiday period (December), there is usually a decline in shopping activity. Many consumers may have spent their budget during the holidays and tend to spend less in January and February. In the August-October period, it is the end of summer, so there is likely to be a decline in sales of seasonal items such as summer clothing and holiday equipment. To overcome the negative impact on profit, Adidas can offer discounts to consumers so that consumers are more confident in buying products because the prices are cheaper, and it can increase units sold and ultimately, increase profits compared to without discounts.

The line chart can also be seen that on average in 2020, Adidas' operating profit was lower than in 2021. This is because other significant factors affect Adidas' sales, such as the COVID pandemic in the US (2020). Peak sales can be seen in July 2021 and December 2021, for those months Adidas can ensure that products with high demand have good inventory to anticipate the peak sales period. The lowest sales can be seen in January-early March, Adidas can consider promoting items that are in line with the current season or upcoming trends and reducing inventory of low-demand products in order to save inventory costs and increase overall profits.

## More units sold, higher operating profit for different product categories?
    - Data Visualisation: Bubble Chart
    - Columns: product, units sold, operating profit
    - Objective: to find out which Adidas product categories have the most sales and to find out if there are other factors that impact profit besides units sold (e.g., lower profit despite many units sold, means the price is too low/discount is too high)

I use bubble chart because it is the most suitable. If I use grouped bar chart, because the value of units sold is too low compared to the value of operating profit, then the bar that is units sold cannot be seen on the chart. To see if there is a direct relationship between units sold and operating profit for various products and to make it easier to compare between units sold and operating profit, I use bubble chart where the bubble size is determined by Units Sold and the color is determined by Operating Profit.

In [10]:
# Group by Product Category and only add values ​​in the Units Sold and Operating Profit columns
df_grouped = df.groupby('Product Category', as_index=False).agg({
 'Units Sold': 'sum',
 'Operating Profit': 'sum'
})

# Create a bubble chart using Plotly Express
fig = px.scatter(df_grouped, x='Product Category', y='Operating Profit', size='Units Sold', color='Operating Profit',
 hover_name='Product Category', title='Adidas Product Categories by Units Sold and Operating Profit',
 size_max=60, color_continuous_scale='Magenta')

fig.update_layout(title_x=0.5)

fig.show()

Men's Street Footwear => highest units sold, highest operating profit
Women's Apparel => lower units sold than Men's Athletic Footwear units sold, higher operating profit than Men's Athletic Footwear operating profit
Women's Athletic Footwear => lowest units sold, lowest operating profit

Based on the information obtained from the bubble chart, Men's Street Footwear has the highest units sold and the highest operating profit. So, instead of allocating resources to produce more Women's Athletic Footwear (it is the product category that generates the lowest operating profit), Adidas can allocate resources to produce more Men's Street Footwear and for example, it can consider launching a new design for Men's Street Footwear to attract consumers.

Also, it can be seen that Women's Apparel units sold are lower than Men's Athletic Footwear units sold but its operating profit is higher than Men's Athletic Footwear operating profit. This could be because the price of Men's Athletic Footwear products is low or the discount is too high for this product category and it can be overcome by increasing the price or reducing the discount to increase revenue in order to increase profit. But there is a risk of increasing the churn rate. This could also indicate that the manufacturing cost for Men's Athletic Footwear is too high, and to overcome this, Adidas can use cheaper alternative raw materials to make Men's Athletic Footwear products.

In addition, for Women's Street Footwear, Men's Apparel and Women's Athletic Footwear, units sold are related to operating profit where higher units sold, higher operating profit.

Overall, more units sold means higher operating profit for various product categories if there are no other factors affecting it (price too low, discount too high, and manufacturing cost too high, etc.)

## Retailers that contribute the most and least to Adidas' operating profit
    - Data Visualisation: Bar Chart
    - Columns: retailer, operating profit
    - Purpose: to find out which retailers contribute the most and the least to Adidas' profit, so that Adidas can allocate more products to retailers that contribute significantly to Adidas' profit and for retailers that contribute low to profit, Adidas' management team can find out the reasons for poor performance and consider strategies to increase profit.

Since we want to compare the total operating profit generated by various retailers (2020-2021), a bar chart is suitable for visualizing it. A line chart is not suitable for this problem because this question is not about looking at trends over time.

In [11]:
# Group by Retailer and only add values ​​in the Operating Profit column
df_grouped = df.groupby('Retailer', as_index=False).agg({
 'Operating Profit': 'sum'
})


# Create a bar chart using Plotly Express
fig = px.bar(df_grouped, x='Retailer', y='Operating Profit',
 title='Operating Profit by Retailer',
 color='Operating Profit',
 color_continuous_scale='Magenta')

fig.update_layout(title_x=0.5)

fig.show()

As can be seen, West Gear retailers contribute the most to Adidas’ operating profit and Walmart contributes the least to Adidas’ operating profit. Adidas can divert more resources and attention to retailers that show good performance such as West Gear to maximize profitability. Also, for retailers that show poor performance such as Walmart, Adidas can look for opportunities to partner with other retailers that may contribute better profit than Walmart so that Adidas can achieve its goal of increasing operating profit by 150% in one year.

## Find the correlation value of units sold and costs of goods sold (COGS)
    - Descriptive Statistics: Spearman Correlation (Skewed)
    - Column: units sold, COGS (costs of goods sold)
    - Purpose: to check whether operational, material, labor costs and transportation costs, etc. are high. If so, can suggest Adidas a strategy to lower the COGS

Because to analyze profit, it is not only by analyzing the units sold factor, but other significant factors such as Costs of Goods Sold (COGS) can affect operating profit, so I will create a new column 'COGS' which reduces 'Operating Profit' from 'Total Sales'.

In [12]:
# Create a 'COGS' column by subtracting 'Operating Profit' from 'Total Sales'
df['COGS'] = df['Total Sales'] - df['Operating Profit']
df

Retailer Invoice Date          State           Product Category  \
0     Foot Locker   2020-01-01       New York      Men's Street Footwear   
1     Foot Locker   2020-01-02       New York    Men's Athletic Footwear   
2     Foot Locker   2020-01-03       New York    Women's Street Footwear   
3     Foot Locker   2020-01-04       New York  Women's Athletic Footwear   
4     Foot Locker   2020-01-05       New York              Men's Apparel   
...           ...          ...            ...                        ...   
9643  Foot Locker   2021-01-24  New Hampshire              Men's Apparel   
9644  Foot Locker   2021-01-24  New Hampshire            Women's Apparel   
9645  Foot Locker   2021-02-22  New Hampshire      Men's Street Footwear   
9646  Foot Locker   2021-02-22  New Hampshire    Men's Athletic Footwear   
9647  Foot Locker   2021-02-22  New Hampshire    Women's Street Footwear   

      Price per Unit  Units Sold  Total Sales  Operating Profit       COGS  
0                 50        1200     600000.0         300000.00  300000.00  
1                 50        1000     500000.0         150000.00  350000.00  
2                 40        1000     400000.0         140000.00  260000.00  
3                 45         850     382500.0         133875.00  248625.00  
4                 60         900     540000.0         162000.00  378000.00  
...              ...         ...          ...               ...        ...  
9643              50          64       3200.0            896.00    2304.00  
9644              41         105       4305.0           1377.60    2927.40  
9645              41         184       7544.0           2791.28    4752.72  
9646              42          70       2940.0           1234.80    1705.20  
9647              29          83       2407.0            649.89    1757.11  

[9648 rows x 9 columns]

Before checking the correlation value of units sold and COGS between various products, I have to find the skewness and kurtosis values ​​to check whether the distribution between units sold and COGS is normal or not so that I know whether to use Pearson or Spearman correlation.

In [13]:
# Skewness of the Units Sold and COGS distribution
us_skew = df['Units Sold'].skew()
cogs_skew = df['COGS'].skew()
print("Skewness Units Sold:", us_skew)
print("COGS Skewness:",cogs_skew)

Skewness Units Sold: 1.4622572136049727
COGS Skewness: 1.9639431803612994


Based on the skewness information above, the skewness of units sold (1.46) and COGS (1.96) because > 0.05, Units Sold and COGS have a skewed distribution. Therefore, Spearman correlation will be used.

In [14]:
# Import scipy for correlation analysis
from scipy import stats

# Calculate correlation using Spearman's rank correlation
corr_rho, _ = stats.spearmanr(df['Units Sold'], df['COGS'])

print(f"Correlation between units sold and COGS: {corr_rho}")

Correlation between units sold and COGS: 0.9126463908887222


As can be seen, there is a very strong negative correlation between Units Sold and COGS (0.91), this indicates that if units sold at Adidas increase, then COGS will increase. This shows that high sales volume is associated with higher costs of goods sold. This indicates that higher units sold higher COGS due to higher costs such as inventory, manufacturing, transportation, etc.

So to lower COGS while selling more products, there are several strategies that Adidas can implement:

1. If increased sales significantly increase overall costs, Adidas can replace materials used to make products with cheaper alternative materials or cut production on items that receive lower sales

2. Adidas can adopt automation technology in the production process can reduce labor costs and increase efficiency in the long run because it will do repetitive manufacturing tasks

## Find the correlation value between price per unit and units sold

Before checking the correlation value of units sold and price per unit, the skewness and kurtosis values ​​must be sought to check whether the distribution between units sold and price per unit is normal or not in order to know whether to use Pearson or Spearman correlation.

In [15]:
# Skewness of the distribution of Units Sold and Price per Unit
us_skew = df['Units Sold'].skew()
ppu_skew = df['Price per Unit'].skew()
print("Skewness Units Sold:", us_skew)
print("Skewness Price per Unit:",ppu_skew)

Skewness Units Sold: 1.4622572136049727
Skewness Price per Unit: 0.3624454995891895


Based on the skewness information above, the skewness of units sold (1.46) and price per unit (0.36) because > 0.05, Units Sold and Price per Unit have a skewed distribution. Therefore, Spearman correlation will be used.

In [16]:
# Calculate correlation using Spearman's rank correlation
corr_rho, _ = stats.spearmanr(df['Units Sold'], df['Price per Unit'])

print(f"Correlation between units sold and price per unit: {corr_rho}")

Correlation between units sold and price per unit: 0.25859577400998174


As can be seen, there is a weak positive correlation between Units Sold and Price per Unit (0.26), this indicates that sometimes, if the price of a product at Adidas increases, then the units sold also increase. Usually, if the price of a product increases, the units sold should decrease. This suggests that if Adidas has some premium products, a higher price may be considered a sign of better quality, resulting in higher sales. In addition, the reason why the units sold increase is also because Adidas already has a strong loyalty program, so consumers still buy Adidas products even though the price increases.

So to make the price per unit not affect the units sold, Adidas can strengthen the loyalty program such as special discounts for members so that consumers continue to buy Adidas products even though Adidas increases the price of its products.

However, because the correlation is weak, this means that the relationship between price and sales is inconsistent and other factors may have a greater influence on sales.

## Is there a significant difference in the average price per unit between high-selling and low-selling products?

Because the skewness of price per unit sold and units sold is a skew distribution with a p-value above 0.05, it is more suitable to use a non-parametric test such as the Mann-Whitney U test because it does not assume normality.

The Mann-Whitney U test can compare the differences between two independent groups and the dependent variable is continuous. In this context, the two independent groups are Units Sold because it is used to divide the data into two groups: a group with higher sales and a group with lower sales while the dependent variable is Price per Unit because this test tests whether the Price per Unit is significantly different between these two groups.

**H0: The distribution of prices per unit of products with high sales and low sales is the same and there is no difference in the central tendency of prices per unit between products with high sales and products with low sales.**

**H1: The distribution of prices per unit of products with high sales and low sales is different, there is a difference in the central tendency of prices per unit between products with high sales and products with low sales**

In [17]:
# Import from statistics to perform the Mann-Whitney U test
from scipy.stats import mannwhitneyu

# Group data based on median units sold
median_units_sold = df['Units Sold'].median()
high_sales = df[df['Units Sold'] > median_units_sold]['Price per Unit']
low_sales = df[df['Units Sold'] <= median_units_sold]['Price per Unit']

# Using Mann-Whitney U test
u_stat, p_value = mannwhitneyu(high_sales, low_sales)
print(f"U-statistic: {u_stat}, P-value: {p_value}")

U-statistic: 14352404.5, P-value: 6.435288396844111e-88


To check whether this hypothesis is true or not, the median price per unit can be checked.

In [18]:
median_high_sales = high_sales.median()
median_low_sales = low_sales.median()
print("Median price per unit for high sales:", median_high_sales)
print("Median price per unit for low sales:", median_low_sales)

Median price per unit for high sales: 49.0
Median price per unit for low sales: 42.0


Based on the median value, the product prices have quite significant differences, so this hypothesis test is valid.

Because the p-value obtained is very small and <0.05, the null hypothesis is rejected and the alternative hypothesis H1 is accepted. This means that there is a significant difference between the price per unit of products included in the high sales category and products included in the low sales category. This shows that products with premium prices can have higher sales because they are considered to be of higher quality or desired by consumers, while for products with low sales, the products are of lower quality and the price per unit is not attractive enough for consumers. It can also be said that the product is not as competitive as other products with higher sales.

Based on the information above, there are several strategies that Adidas can do:
- To increase Adidas' profit, the product price management team can evaluate product prices and survey loyal consumers about the prices they consider reasonable and willing to pay in order to increase the price of competitive Adidas products and included in the high sales category within a certain range so as not to cause churn
- To increase the pricing competition of Adidas products included in the low sales category, Adidas can research the preferences that are currently trending and create products based on those preferences or consider lowering the price per unit for products with low sales to encourage sales

# 5) Conclusion

Adidas can maximize profit by changing its pricing strategy:
- because the correlation between price per unit and units sold is weak, there is no significant relationship between price per unit and units sold.
- because there is also a significant difference between the price per unit of products with high and low units sold, where products with higher units sold are more competitive than products with lower units sold
- these two insights show that Adidas can:
    - increase profit by increasing the price of more competitive products
    - hold discounts for less competitive products to encourage sales.
    - if increasing the price of the product can increase the churn rate:
        - this negative impact can be minimized by Adidas strengthening its loyalty program (special discounts for members) so that consumers continue to buy Adidas products.

For resource allocation and production, Adidas can:
- focus production and resource allocation and products on its retailers located in high-selling locations, such as New York
- hold discounts or advertise products more to drive demand for low-selling locations

Adidas must also pay attention to its profit trends in certain periods:
- In March-April (Easter/April Fools'), June-July (Summer Break), where Adidas's profit is highest, Adidas can launch marketing campaigns targeted for certain celebrations or seasons.
- In January-February and the August-October period, because consumers have spent their budget during the holidays, Adidas can offer discounts to consumers so that consumers are more confident in buying products because the prices are cheaper.

Adidas must also pay attention to the demand for various product categories:
- Based on the analysis of units sold and operating profit, it can also be seen that the price of Men's Athletic Footwear must also be increased or discounts lowered to increase Adidas's profits
- Adidas can also allocate resources to produce more Men's Street Footwear because the demand for this product category is the highest.

Adidas should also pay attention to the performance of retailers from operating profit, and carry out the following strategies:
- divert more resources and attention to retailers that show good performance such as West Gear to maximize profitability.
- for retailers that show poor performance such as Walmart, Adidas can look for opportunities to partner with other retailers that may provide better profit contributions than Walmart so that Adidas can achieve the goal of increasing operating profit by 150% in one year